library/alloc/src/vec/mod.rs

```diff
@@ -4032,3 +4032,47 @@ impl<T, A: Allocator, const N: usize> TryFrom<Vec<T, A>> for [T; N] {
         Ok(array)
     }
 }
+
+#[cfg(kani)]
+#[unstable(feature = "kani", issue = "none")]
+mod verify {
+    use core::kani;
+    use crate::vec::Vec;
+
+    #[kani::proof]
+    pub fn verify_swap_remove() {
+        let mut vec = Vec::new();
+        // Creating an empty vector and then populating it with three arbitrary integers
+        for _ in 0..3 {
```

Comment:
Can we hardcode the length as it's a magic number and have a comment explaining it?

Reply:
Okay